src/main/java/ru/yandex/qatools/embed/postgresql/store/PostgresArtifactStore.java

```diff
@@ -60,14 +62,6 @@ public void removeFileSet(Distribution distribution, IExtractedFileSet all) {
         }
     }
 
-    @Override
-    public boolean checkDistribution(Distribution distribution) throws IOException {
-        if (!LocalArtifactStore.checkArtifact(downloadConfig, distribution)) {
```

Comment:
It looked to me the only reason you used the flapdoodle package was to use the LocalArtifactStore. It seemed to me though that this method was identical to the one in the flapdoodle ArtificatStore class. This means we can just extend the ArtifactStore and override the methods we want to override and remove the need to override the checkDistribution method. This allows us to remove the usage of the flapdoodle package and solve the split package problem.

Reply:
@smecsia it looks like all the checks have passed for the pull request. Are you able to review it and merge or offer suggestions? Thanks in advance!